examples/Optimization_Example.ipynb

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
Does this need an "Open in Colab" button?

Reply:
It could, what is the standard on those? (I've seen in notebook, in a readme file, both, neither)

Reply:
I don't have a rule of thumb but I know it's on the Simple Example and it's great. Let's just add to `.ipynb` files?